<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QFDSF5H" name="Default test" tests="1" failures="1" timestamp="23 Apr 2016 12:51:16 GMT" time="73.19" errors="0">
  <testcase name="uploadFile" time="65.392" classname="com.optyle.dropboxTask.Tests.DropDownTests">
    <failure type="java.lang.IllegalArgumentException" message="&amp;apos;targetPath&amp;apos;: bad path: must start with &amp;quot;/&amp;quot;: &amp;quot;C:\\Users\\asus\\Documents\\workspace\\DropboxAutomation\\Worter.txt&amp;quot;">
      <![CDATA[java.lang.IllegalArgumentException: 'targetPath': bad path: must start with "/": "C:\\Users\\asus\\Documents\\workspace\\DropboxAutomation\\Worter.txt"
	at com.dropbox.core.DbxPath.checkArg(DbxPath.java:48)
	at com.dropbox.core.DbxClient.startUploadFileSingle(DbxClient.java:652)
	at com.dropbox.core.DbxClient.startUploadFile(DbxClient.java:619)
	at com.dropbox.core.DbxClient.uploadFile(DbxClient.java:561)
	at com.dropbox.core.DbxClient.uploadFile(DbxClient.java:514)
	at com.optile.dropbox.utilities.Utilities.uploadFile(Utilities.java:21)
	at com.optyle.dropboxTask.Tests.DropDownTests.uploadFile(DropDownTests.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- uploadFile -->
</testsuite> <!-- Default test -->
